crates/nu-utils/src/sample_config/default_config.nu

```diff
@@ -167,6 +167,10 @@ $env.config = {
         header_on_separator: false # show header text on separator/border line
     }
 
+    errors: {
+        style: "fancy" # "fancy" or "narratable" for screen reader-friendly error messages
```

Comment:
Unless we have more error configs, I would not nest and instead use `error_style: "fancy"`.

Reply:
Just moved it outside to a single option in 281c1afc3c713ecb68ed06daafcafdfd17742cb0.

I noticed that some config options are reconstructed after failing validation, while others (specially more simple fields like strings) aren't. I still do the reconstruction for this option according to the rules™️ 

```rust
// Config record (self) mutation rules:
// * When parsing a config Record, if a config key error occurs, remove the key.
// * When parsing a config Record, if a config value error occurs, replace the value
// with a reconstructed Nu value for the current (unaltered) configuration for that setting.
// For instance:
// $env.config.ls.use_ls_colors = 2 results in an error, so
// the current use_ls_colors config setting is converted to a Value::Boolean and inserted in the
// record in place of the 2.
```

Happy to change it if I misunderstood :)